lib/galaxy/webapps/galaxy/api/genomes.py

```diff
@@ -94,18 +98,17 @@ def show(
 
     @router.get(
         "/api/genomes/{id}/indexes",
-        summary="Return all available indexes for a genome id for provided type",
-        response_description="Indexes for a genome id for provided type",
+        summary="Return all available indexes for a genome id",
+        response_description="Indexes for a genome id",
     )
     def indexes(
         self,
-        trans: ProvidesUserContext = DependsOnTrans,  # may want to get custom index in the future
+        trans: ProvidesUserContext = DependsOnTrans,
         id: str = IdPathParam,
-        type: str = IndexTypeQueryParam,
```

Comment:
Can you leave it and just say it's always fasta_indexes ? This would be a potential breaking change.